java/java_test.go

```diff
@@ -376,6 +376,44 @@ func testJavaWithBuilder(builder string) func(*testing.T, spec.G, spec.S) {
 				})
 			})
 
+			context("app uses opentelemetry", func() {
+				it("builds successfully", func() {
+					var err error
+					source, err = occam.Source(filepath.Join("../java", "opentelemetry"))
+					Expect(err).NotTo(HaveOccurred())
+
+					var logs fmt.Stringer
+					image, logs, err = pack.Build.
+						WithPullPolicy("never").
```

Comment:
```suggestion
						WithPullPolicy("if-not-present").
```

Needs to be "if-not-present" since we're utilizing a buildpack that is not present in the default Java composite or the builder we are using.

Reply:
Updated as suggested